eul/Store/BatteryStore.swift

```diff
@@ -52,5 +56,6 @@ class BatteryStore: ObservableObject, Refreshable {
 
     init() {
         initObserver(for: .StoreShouldRefresh)
+        refresh()
```

Comment:
i think `.StoreShouldRefresh` will be called once the app finished launching, see `AppDelegate.swift:53`

Reply:
The call of `.StoreShouldRefresh` seems to be slower, and he does not guarantee that the `valid` is correct when using `BatteryStore`. Calling refresh manually is only one way to solve the problem, there may be a better way.

Reply:
reasonable. it's cool to keep as it is for now